modules/books.js

```diff
@@ -0,0 +1,45 @@
+export default class Book {
+  constructor() {
+    this.bookStore = JSON.parse(localStorage.getItem('singleBook')) || [];
+  }
+
+    getBookList = () => {
+      const bookList = document.getElementById('book-list');
+      bookList.innerHTML = '';
+      bookList.innerHTML += this.bookStore.map(
+        (el, index) => ` <div class="${index % 2 === 0 ? 'dark' : 'mediumdark'}">
+          <h4>${el.title}</h4>
+          <p>${el.author}</p>
+          <button type="button" id=${el.id} class='remove-button'>Remove</button>
+          </div>`,
+      );
```

Comment:
- [ ] OPTIONAL - Please, consider that this `comma` is not needed - try to delete it from line 14, after the closing `div` tag - your UI will look more professional.
<img width="799" alt="Screen Shot 2022-05-03 at 23 22 37" src="https://user-images.githubusercontent.com/49064106/166541922-2d60e380-eb7b-4b70-8ad8-a82fd7ef908b.png">

Reply:
Ohh thank you, I am going to do that